torch/onnx/_internal/fx/serialization.py

```diff
@@ -38,8 +39,8 @@ def _create_tensor_proto_with_external_data(
 
     tensor_proto = onnx.TensorProto()
     tensor_proto.name = name
-    tensor_proto.data_type = torch.onnx._type_utils._SCALAR_TYPE_TO_ONNX[  # type: ignore[assignment]
-        torch.onnx._type_utils._DTYPE_TO_SCALAR_TYPE[tensor.dtype]
+    tensor_proto.data_type = _type_utils._SCALAR_TYPE_TO_ONNX[  # type: ignore[assignment]
+        _type_utils._DTYPE_TO_SCALAR_TYPE[tensor.dtype]
```

Comment:
Looks like we didn’t catch this in previous PRs, but we shouldn’t use these private fields. They are by definition private. Possible to use the public methods instead?

Reply:
Done